test/spec/cloud-element-templates/properties/BooleanProperty.spec.js

```diff
@@ -58,6 +58,34 @@ describe('provider/cloud-element-templates - BooleanProperty', function() {
   }));
 
 
+  describe('property', function() {
+
+    let entry, input;
+
+    beforeEach(async function() {
```

Comment:
I'd ensure that we also have tests to verify this against the element templates API, applying a template.

Reply:
Done via https://github.com/bpmn-io/bpmn-js-element-templates/pull/161/commits/f1fe6a749ba025450465f274be942339e9881844